Mathlib/CategoryTheory/Sites/Sheafification.lean

```diff
@@ -166,14 +166,15 @@ def mk {X : C} {P : Cᵒᵖ ⥤ D} {S : J.Cover X} (x : Meq P S) : (J.plusObj P)
 theorem res_mk_eq_mk_pullback {Y X : C} {P : Cᵒᵖ ⥤ D} {S : J.Cover X} (x : Meq P S) (f : Y ⟶ X) :
     (J.plusObj P).map f.op (mk x) = mk (x.pullback f) := by
   dsimp [mk, plusObj]
-  simp only [← comp_apply, colimit.ι_pre, ι_colimMap_assoc]
+  erw [← comp_apply, ι_colimMap_assoc, colimit.ι_pre]
```

Comment:
```suggestion
  erw [← comp_apply]
  rw [ι_colimMap_assoc, colimit.ι_pre]
```
I think it's nice to minimize the span during which we need to resort to `erw`. It costs a few keystrokes but typically every `erw` is a sign things aren't quite right, so I'd prefer to make it as explicit as possible.

(Obviously mathlib does not do this consistently, so feel free to ignore this one if you disagree.)

Reply:
I agree, let me try if I can make this `erw` go away even